app/uk/gov/hmrc/tai/binders/TaxYearObjectBinder.scala

```diff
@@ -28,22 +26,21 @@ object TaxYearObjectBinder {
     override def bind(key: String, value: String): Either[String, TaxYear] =
       try {
 
-        lazy val appConfig = Play.current.injector.instanceOf[ApplicationConfig]
-
         val taxYearRequired: TaxYear = TaxYear(value)
 
         val currentYear: Int = TaxYear().year
-
         val latestSupportedTaxYear = TaxYear().next
-        val earliestSupportedTaxYear = TaxYear(currentYear - appConfig.numberOfPreviousYearsToShow)
+
+        val numberOfPreviousYearsToShow = 5
```

Comment:
Isn't this a magic number? Shouldn't this be stored in the config file to make it easier to find if it's changed?

Reply:
If you check the next comment, there's already a value stored in the ApplicationConfig that you can use

Reply:
Can't inject appConfig so hard coded here